core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -31,6 +31,7 @@ oppia.controller('StateHints', [
     $scope.EditorStateService = EditorStateService;
     $scope.stateHintsService = stateHintsService;
     $scope.activeHintIndex = null;
+    $scope.hintCount = 0;
```

Comment:
Don't do this -- it's a nice idea, but keeping up-to-date in the future will be a pain. See my suggestion in the HTML file.

Reply:
So, I'll remove hintCount altogether and use stateHintsService.displayed.length for the checks?